slipp/src/main/java/slipp/support/db/ConnectionManager.java

```diff
@@ -1,31 +1,41 @@
 package slipp.support.db;
 
 import org.apache.commons.dbcp2.BasicDataSource;
+import slipp.support.db.exception.LoadPropertyFailException;
 
 import javax.sql.DataSource;
-import java.sql.Connection;
-import java.sql.SQLException;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
 
 public class ConnectionManager {
-    private static final String DB_DRIVER = "org.h2.Driver";
-    private static final String DB_URL = "jdbc:h2:mem:jwp-framework";
-    private static final String DB_USERNAME = "sa";
-    private static final String DB_PW = "";
+
+    private static final String DEFAULT_PROPS_PATH = "./slipp/src/main/resources/db.properties";
```

Comment:
```suggestion
    private static final String DEFAULT_PROPS_PATH = "src/main/resources/db.properties";
```

이게 맞지안나요.
그리고 다른 모듈에서는 다른 외부 설정파일을 로드할 수 있도록 작성해보는 것은 어떨까요